lambda/index.ts

```diff
@@ -7,7 +7,7 @@ export async function processor(event: any): Promise<any> {
   for await (const record of event.Records) {
     try {
       const requestBody = JSON.parse(JSON.parse(record.body).Message);
-      console.log(`message received -> ${JSON.stringify(requestBody)}`);
+      console.log(`message received-> ${JSON.stringify(requestBody)}`);
```

Comment:
just to retrigger lambda deploy in prod - as the current prod env is not pulling in the ecs endpoint env variable